updater/cmd/updater/main.go

```diff
@@ -40,7 +40,7 @@ func main() {
 
 	// We can't create the zap logger yet, because we don't know the install dir, which is needed
 	// to create the logger. So we pass a Nop logger here.
-	installDir, err := path.InstallDir(zap.NewNop())
+	installDir, err := path.InstallDir(zap.NewNop(), path.DefaultConfigOverrides)
```

Comment:
path.InstallDir takes a list of package overrides. On Linux this will be `/etc/default/observiq-otel-collector` and `/etc/sysconfig/observiq-otel-collector`. On Windows and Darwin, this option is ignored.